{{cookiecutter.project_dirname}}/requirements/common.in

```diff
@@ -1,2 +1,5 @@
 # pip-compile --output-file common.txt common.in
 -r base.in
+djangorestframework-camel-case~=1.3.0
+djangorestframework~=3.13.0
+drf-spectacular~=0.22.0
```

Comment:
Let's discuss what exactly we should ask the user for, with the aim to make DRF an optional dependency.